File: CrossFiles/PHP2/Class1.php
<?php

class Class1 {
    private $input;

    public function __construct($input) {
        $this->input = $input;
    }

    public function process() {
        echo "Class1 processing: {$this->input}\n";
        // Potentially unsafe operation
        system("echo " . $this->input);
    }
}
?>

File: CrossFiles/Java/src/main/java/Class1.java
import java.io.IOException;

public class Class1 {
    private String input;

    public Class1(String input) {
        this.input = input;
    }

    public void process() {
        System.out.println("Class1 processing: " + input);
        // Potentially unsafe operation
        try {
            Runtime.getRuntime().exec("echo " + input);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}
